### PR TITLE
o16n: primitives code location

### PR DESCRIPTION
**Pull request by alxhub**
Nothing to see here, Enea

### CODE REVIEW

File: packages/core/src/render3/reactivity/signal.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   * Update the value of the signal based on itfs current value, and
+   * notify any dependents.
+   */
+  update(updateFn: (value: T) => T): void;
+
+  /**
+   * Update the current value by mutating it in-place, and
+   * notify any dependents.
+   */
+  mutate(mutatorFn: (value: T) => void): void;
```
Comment (eneajaho) [inline packages/core/src/render3/reactivity/signal.ts:34]: So we are keeping the mutate fn? https://github.com/angular/angular/pull/51821

File: packages/core/src/render3/reactivity/signal.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+ * @developerPreview
+ */
+export interface WritableSignal<T> extends Signal<T> {
+  /**
+   * Directly set the signal to a new value, and notify any dependents.
+   */
+  set(value: T): void;
+
+  /**
+   * Update the value of the signal based on itfs current value, and
```
Comment (rhutchison) [inline packages/core/src/render3/reactivity/signal.ts:25]: itfs?

File: packages/core/src/render3/reactivity/signal.ts (hunk 1 above)
Comment (pkozlowski-opensource) [reply packages/core/src/render3/reactivity/signal.ts:34]: Probably not :-)

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: public-api 
Reviewed-for: fw-core

**Comment by alxhub** [comment]
Caretaker: This will need cl/571389475 in the sync.

**Comment by alxhub** [comment]
Caretaker:

* pullapprove disabled (code was already approved, partially in another PR)
* this change requires a g3 patch: cl/571389475
* the CI failure is due to the symbol test for defer which is already fixed

**Comment by atscott** [comment]
This PR was merged into the repository by commit 00128e38538f12fe9bc72ede9b55149e0be5ead0.

**Pull request closed without merging**